### PR TITLE
Fixed error in search function

### PR DESCRIPTION
**Pull request by NoMore201**
In the block of code accessed with the verbose flag, there is a wrong
conversion of a string variable, ("%s" % item) into a byte one, giving
error when trying to strip() it.

Signed-off-by: Domenico Iezzi <domenico.iezzi.201@gmail.com>

### CODE REVIEW

**Comment by NoMore201** [comment]
Now the library should be fully working on python3. The only thing is that the token dispenser is not working at the moment, so the builds will fail.

File: ext_libs/googleplay_api/googleplay.py [hunk 1]
```diff
@@ -184,7 +193,7 @@ def executeRequestApi2(self, path, datapost=None, post_content_type="application
         self._try_register_preFetch(message)
 
         # Debug
-        #print text_format.MessageToString(message)
+        print(text_format.MessageToString(message))
```
Comment (matlink) [inline ext_libs/googleplay_api/googleplay.py]: Why always enable debug?

**Comment by NoMore201** [comment]
Sorry for the debug string, I forgot to comment it out. As for the travis-ci build, python3 build is passing. python2 build is still failing because of pyaxmlparser which is python3-only

**Comment by matlink** [comment]
Well, I'll need to drop the 2.7 support when 3.x will work great. So in the python3support branch, we should remove the 2.7 compilation for travis.

**Pull request merged**